content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -175,6 +175,24 @@ at install time and cannot be changed once the install is running.
 
 The decision you make will be entered during setup.
 
+#### Mounted Disk Guidelines
+
+The mounted disk option provides for significant flexibility in how PTFE stores it's data. In order to help customers narrow down possibilites we're providing some guidelines about how to use mounted disk.
```

Comment:
it's data > its data
In order to > To
~customers~
we're providing... > here are our guidelines about using mounted disk storage.